qa/common/src/main/java/org/eclipse/kapua/qa/common/utils/InitTestEnvironment.java

```diff
@@ -12,24 +12,13 @@
  *******************************************************************************/
 package org.eclipse.kapua.qa.common.utils;
 
-import org.eclipse.kapua.KapuaException;
-import org.eclipse.kapua.commons.util.xml.XmlUtil;
-import org.eclipse.kapua.qa.common.TestJAXBContextProvider;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
 import io.cucumber.guice.ScenarioScoped;
 import io.cucumber.java.en.Given;
 
 @ScenarioScoped
-public class InitJaxbContext {
-
```

Comment:
JAXB no longer needs to be initialized within each test as a separated step. However in a few tests it was the only step referenced in context initialization - which also performs other steps. As such, a placeholder step was needed in its stead.